llvm/lib/DWARFLinker/Classic/CMakeLists.txt

```diff
@@ -1,11 +1,11 @@
-add_llvm_component_library(LLVMDWARFLinker
+add_llvm_component_library(LLVMDWARFLinkerClassic
   DWARFLinkerCompileUnit.cpp
   DWARFLinkerDeclContext.cpp
   DWARFLinker.cpp
   DWARFStreamer.cpp
 
   ADDITIONAL_HEADER_DIRS
-  ${LLVM_MAIN_INCLUDE_DIR}/llvm/DWARFLinker
+  ${LLVM_MAIN_INCLUDE_DIR}/llvm/DWARFLinker/Classic
```

Comment:
unrelated; maybe this line isn't needed at all?

Reply:
Ok, will change this and next comment with a separate patch.